release/src/main/scripts/run_rc_validation.sh

```diff
@@ -36,223 +51,186 @@ function clean_up(){
-  echo "[GCP Project Required] Please input your GCP project:"
-  read USER_GCP_PROJECT
-  echo "[GCP GCS Bucket Required] Please input your GCS bucket: "
-  read USER_GCS_BUCKET
-  MOBILE_GAME_DATASET=${USER}_java_validations
-  MOBILE_GAME_PUBSUB_TOPIC=leader_board-${USER}-java-topic-1
-  echo "Please review following GCP sources setup: "
+echo ""
+echo "====================Starting Java Mobile Game====================="
+if [[ "$java_mobile_game" = true ]]; then
+  MOBILE_GAME_DATASET=${USER}_java_validations_$(date +%m%d)_$RANDOM
+  MOBILE_GAME_PUBSUB_TOPIC=leader_board-${USER}-java-topic-$(date +%m%d)_$RANDOM
   echo "Using GCP project: ${USER_GCP_PROJECT}"
   echo "Will create BigQuery dataset: ${MOBILE_GAME_DATASET}"
   echo "Will create Pubsub topic: ${MOBILE_GAME_PUBSUB_TOPIC}"
-  echo "[Confirmation Required] Do you want to run validations with configurations above? [y|N]"
-  read confirmation
-  if [[ $confirmation = "y" ]]; then
-    gcloud auth login
-    gcloud config set project ${USER_GCP_PROJECT}
-    echo "-----------------Setting Up Service Account------------------------"
-    echo "Please go to GCP IAM console under your project(${USER_GCP_PROJECT})."
-    echo "Create a service account as project owner, if you don't have one."
-    echo "[Input Required] Please enter your service account email:"
-    read USER_SERVICE_ACCOUNT_EMAIL
-    SERVICE_ACCOUNT_KEY_JSON=${USER}_json_key.json
-    gcloud iam service-accounts keys create ${SERVICE_ACCOUNT_KEY_JSON} --iam-account ${USER_SERVICE_ACCOUNT_EMAIL}
-    export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/${SERVICE_ACCOUNT_KEY_JSON}
 
-    echo "-------------------Creating BigQuery Dataset-----------------------"
-    bq rm -rf --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
-    bq mk --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
-
-    echo "----------------------Creating Pubsub Topic------------------------"
-    gcloud alpha pubsub topics delete projects/${USER_GCP_PROJECT}/topics/${MOBILE_GAME_PUBSUB_TOPIC}
-    gcloud alpha pubsub topics create --project=${USER_GCP_PROJECT} ${MOBILE_GAME_PUBSUB_TOPIC}
-
-    echo "**************************************************************************"
-    echo "* Java mobile game validations: UserScore, HourlyTeamScore, Leaderboard"
-    echo "**************************************************************************"
-    ./gradlew :runners:google-cloud-dataflow-java:runMobileGamingJavaDataflow \
-    -Prepourl=${REPO_URL} \
-    -Pver=${RELEASE} \
-    -PgcpProject=${USER_GCP_PROJECT} \
-    -PgcsBucket=${USER_GCS_BUCKET} \
-    -PbqDataset=${MOBILE_GAME_DATASET} -PpubsubTopic=${MOBILE_GAME_PUBSUB_TOPIC}
-  fi
+  echo "-----------------Creating BigQuery Dataset-----------------"
+  bq mk --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
+
+  echo "-----------------Creating Pubsub Topic-----------------"
+  gcloud alpha pubsub topics create --project=${USER_GCP_PROJECT} ${MOBILE_GAME_PUBSUB_TOPIC}
```

Comment:
Does this need to be "gcloud alpha", or can "alpha" be dropped? https://cloud.google.com/sdk/gcloud/reference/pubsub/topics/create

Reply:
good catch! I removed `alpha` in later commit.